doc/source/citation.md

```diff
@@ -0,0 +1,114 @@
+(citation)=
+
+# Citing and method overview
+
+When using a method implemented in xDEM, one should cite both the package and the original study behind the method (if there is any)!
+
+## Citing xDEM
+
+To cite the package, use the Zenodo DOI: [![Zenodo](https://zenodo.org/badge/doi/10.5281/zenodo.4809697.svg)](https://zenodo.org/doi/10.5281/zenodo.4809697).
+
+## Method overview
+
+For citation and other purposes, here's an overview of all methods implemented in the package and their reference, if it exists.
+More details are available on each feature page!
+
+### Terrain attributes
+
+```{list-table}
+   :widths: 1 1
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Method
+     - Reference
+   * - Slope, aspect and hillshade
+     - [Horn (1981)](http://dx.doi.org/10.1109/PROC.1981.11918) or [Zevenbergen and Thorne (1987)](http://dx.doi.org/10.1002/esp.3290120107)
+   * - Curvatures
+     - [Zevenbergen and Thorne (1987)](http://dx.doi.org/10.1002/esp.3290120107)
+   * - Topographic position index
+     - [Weiss (2001)](http://www.jennessent.com/downloads/TPI-poster-TNC_18x22.pdf)
+   * - Terrain ruggedness index
+     - [Riley et al. (1999)](http://download.osgeo.org/qgis/doc/reference-docs/Terrain_Ruggedness_Index.pdf) or [Wilson et al. (2007)](http://dx.doi.org/10.1080/01490410701295962)
+   * - Roughness
+     - [Dartnell (2000)](https://environment.sfsu.edu/node/11292)
+   * - Rugosity
+     - [Jenness (2004)](<https://doi.org/10.2193/0091-7648(2004)032[0829:CLSAFD]2.0.CO;2>)
+   * - Fractal roughness
+     - [Taud and Parrot (2005)](https://doi.org/10.4000/geomorphologie.622)
+```
+
+### Coregistration
+
+```{list-table}
+   :widths: 1 1
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Method
+     - Reference
+   * - Nuth and Kääb
+     - [Nuth and Kääb (2011)](https://doi.org/10.5194/tc-5-271-2011)
+   * - Dh minimization
+     - N/A
+   * - Iterative closest point
+     - [Besl and McKay (1992)](https://doi.org/10.1117/12.57955)
+   * - Vertical shift
+     - N/A
+```
+
+### Bias-correction
+
+```{list-table}
+   :widths: 1 1
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Method
+     - Reference
+   * - Deramp
+     - N/A
+   * - Directional bias (sum of sinuoids)
+     - [Girod et al. (2017)](https://doi.org/10.3390/rs9070704)
+   * - Directional bias (other)
+     - N/A
+   * - Terrain bias (maximum curvature)
+     - [Gardelle et al. (2012)](https://doi.org/10.3189/2012JoG11J175)
+   * - Terrain bias (elevation)
+     - [Nuth and Kääb (2011)](https://doi.org/10.5194/tc-5-271-2011)
+   * - Terrain bias (other)
+     - N/A
+   * - Vertical shift
+     - N/A
+```
+
+### Gap-filling
+
+```{list-table}
+   :widths: 1 1
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Method
+     - Reference
+   * - Bilinear
+     - N/A
+   * - Local and regional hypsometric
+     - [McNabb et al. (2019)](https://tc.copernicus.org/articles/13/895/2019/)
```

Comment:
It is definitely a good resource to understand the concept of the hypsometric approach, but this is not the paper that actually proposes this method. It has been used for years before Bob's paper. I will check if I can find an original reference for this.

Reply:
McNabb 2019 refers to Arendt (2002) and Kääb (2008). In the first one, there is no occurrence of "hypso". The second one uses the term "hypsography" ! So not sure what is the best way to acknowledge...

Reply:
Good point. Indeed, Anthony is binning per elevation band in his study. I trust @iamdonovan will have done the work on this :stuck_out_tongue:  
We can also check quickly with Anthony: @aaarendt Do you remember if there was another study before your 2002 paper doing "elevation change binning per elevation band" to compute glacier volume change (what we now call hypsometric gap-filling)?